src/richie/apps/core/templates/social-networks/course-badges.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}{% spaceless %}
+{% blocktrans with page_title=page_title page_url=page_url asvar twitter_text %}Je viens de m'inscrire pour '{{ page_title }}' via @funmooc: {{ page_url }}{% endblocktrans %}
+{% blocktrans asvar mailto_subject %}Suivez un cours en ligne avec FUN{% endblocktrans %}
+{% blocktrans with page_title=page_title page_url=page_url asvar mailto_body %}Je viens de m'inscrire pour '{{ page_title }}' via FUN: {{ page_url }}{% endblocktrans %}
+<div class="social-badges">
+    <a href="https://www.facebook.com/france.universite.numerique" target="_blank" class="social-badges__item social-badges__item--facebook" title="{% trans "Facebook page" %}">
+        <span class="icon-facebook">F</span>
```

Comment:
It would be nice to add the icons in the same PR WDYT?

Reply:
yep, just wanted to quickly push something to preview

Reply:
no I juste meant to change the sentence to not mention fun-mooc.
Do exactly what you wrote but replace the text by:
```
I just enrolled to the course "{{ page_title }}" on Richie: {{ page_url }}
```
Replace:
https://www.facebook.com/france.universite.numerique  
by https://www.facebook.com/example  (I checked that it does not exist, so ok...)